### ISSUE TITLE
Perl modules missing POD docs

### ISSUE THREAD

**Issue by ehuelsmann** [ledgersmb/LedgerSMB#1593]
Output on my master-pod-checks branch:
```plain
# LedgerSMB::PriceMatrix: couldn't find pod
# LedgerSMB::Sysconfig: couldn't find pod
# LedgerSMB::Num2text: couldn't find pod
```
~~LedgerSMB::Tax: couldn't find pod~~
~~LedgerSMB::Taxes::Simple: couldn't find pod~~
~~LedgerSMB::DBObject::TransTemplate: couldn't find pod~~
~~LedgerSMB::RESTXML::Document::SalesOrder: couldn't find pod~~
~~LedgerSMB::RESTXML::Document::Part: couldn't find pod~~
~~LedgerSMB::RESTXML::Document::Customer: couldn't find pod~~
~~LedgerSMB::RESTXML::Document::Part_Search: couldn't find pod~~
~~LedgerSMB::Reconciliation::CSV: couldn't find pod~~

**Comment by ehuelsmann** [comment]
@sbts will deal with LedgerSMB::Sysconfig

**Comment by einhverfr** [comment]
I have LedgerSMB::DBObject::TransTemplate,  LedgerSMB::Tax, and LedgerSMB::Taxes::Simple

LedgerSMB::RESTXML::*  are deprecated, might now work (probably don't) and probably should be removed.

**Comment by einhverfr** [comment]
I will also grab the priceatrix module.

**Comment by ehuelsmann** [comment]
That leaves Sysconfig and Num2text...

Maybe we need to declare Num2text "hopelessly part of 'old code'" and "will go away eventually"?

Then, when @einhverfr completes pricematrix and @sbts completes Sysconfig, this issue is completed...

**Comment by ehuelsmann** [comment]
Blacklisted Num2text; added POD to PriceMatrix. Sysconfig comes from sysconfig-reorg branch.

**Comment by ehuelsmann** [comment]
Resolved in 09d234f
